package.json

```
@@ -105,7 +104,10 @@
     "eslint": "6.6.0",
     "eslint-config-airbnb-base": "14.2.0",
     "eslint-plugin-import": "2.22.0",
-    "eslint-plugin-simple-import-sort": "5.0.3"
+    "eslint-plugin-simple-import-sort": "5.0.3",
+    "import-sort-style-module": "^6.0.0",
```

exact version만 사용하면 이견 없습니다

```suggestion
    "import-sort-style-module": "6.0.0",
```